bundles/binding/org.openhab.binding.zwave/database/fibaro/fgs222.xml

```diff
@@ -113,20 +113,23 @@
 			<Type>list</Type>
 			<Default>1</Default>
 			<Size>1</Size>
-			<Label lang="en">Control key #2 behaviour</Label>			
+			<Label lang="en">Sending commands to control devices assigned to 2-nd association group (key no. 2)</Label>			
```

Comment:
I think this is too long for a label? It might screw up the UI. What's wrong with the short label that is already there?